src/mod/applications/mod_http_cache/conf/autoload_configs/http_cache.conf.xml

```diff
@@ -1,47 +1,78 @@
+                <!-- Optional: Backup folder to save uploaded file in case of failure (for example recording a .wav file to a webserver). Must ended without a slash !-->
+                <!--  If you want to use your own s3-compatible service, base domain MUST be set -->
+                <!--<base-domain><![CDATA[stg.vinadata.vn]]></base-domain>-->
+                <!-- Required: AWS region -->
+                <region><![CDATA[ap-southeast-1]]></region>         <!-- base domain is s3-ap-southeast-1.amazonaws.com -->
+                <!-- Optional: Expiration time in seconds for URL signature. Default is 604800s -->
+                <expires>604800</expires>
+            </aws-s3>
 
-  <profiles>
-    <profile name="s3">
-       <!-- Credentials for AWS account. -->
-       <aws-s3>
-          <!-- 20 character key identifier, can override with AWS_ACCESS_KEY_ID environment variable -->
-          <access-key-id><![CDATA[AKIAIOSFODNN7EXAMPLE]]></access-key-id>
-          <!-- 40 character secret, can override with AWS_SECRET_ACCESS_KEY environment variable -->
-          <secret-access-key><![CDATA[wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY]]></secret-access-key>
-          <!--base-url><![CDATA[s3.example.com]]></base-url-->
-       </aws-s3>
-       <!-- Domains that this profile applies to -->
-       <domains>
-          <domain name="bucket.s3.amazonaws.com"/>
-          <domain name="bucket2.s3.amazonaws.com"/>
-       </domains>
-    </profile>
-
-    <profile name="blob">
-      <azure-blob>
-        <!-- key identifier, can override with AZURE_STORAGE_ACCESS_KEY environment variable -->
-        <secret-access-key>kOOY4Y/sqZU9bsLjmN+9McVwTry+UIn1Owt4Zs/2S2FQT0eAWLKsk
-Z0V6/gGFqCAKVvwXoGjqUn7PNbVjhZiNA==</secret-access-key>
-      </azure-blob>
-      <domains>
-        <domain name="account.blob.core.windows.net"/>
-      </domains>
-    </profile>
-  </profiles>
+            <!-- Required: Domains that this profile will automatically be applied. Domains must have the same base domain (s3-ap-southeast-1.amazonaws.com) -->
+            <domains>
+                <domain name="bucket1.s3-ap-southeast-1.amazonaws.com"/>
+                <domain name="bucket2.s3-ap-southeast-1.amazonaws.com"/>
+            </domains>
+        </profile>
+        <profile name="s3-compatible">
+            <!-- Credentials for AWS account or any account on s3-like storage service -->
+            <aws-s3>
+                <!-- Required: Key identifier -->
+                <access-key-id><![CDATA[506665ebbbaffc1701aaf5a61ad88421]]></access-key-id>
+                <!-- Required: Key secret -->
+                <secret-access-key><![CDATA[2dd63d700744e2c1c277be7dc81bfb1b]]></secret-access-key>
+                <!-- Optional: Base domain for the service -->
+                <!-- If you want to use your own s3-compatible service, base domain MUST be set -->
+                <base-domain><![CDATA[stg.vinadata.vn]]></base-domain>
```

Comment:
Use fake example config

Reply:
It's fake config and the domain does not exist.